packages/fastauth-metamask/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@near-wallet-selector/fastauth-metamask",
+  "version": "7.9.2",
+  "description": "Control NEAR accounts with ETH accounts",
+  "author": "mattlockyer",
+  "keywords": [
+    "near",
+    "blockchain",
+    "wallets",
+    "dapps",
+    "near-protocol",
+    "near-blockchain",
+    "wallet selector",
+    "fastauth-metamask"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/near/wallet-selector.git"
+  },
+  "bugs": {
+    "url": "https://github.com/near/wallet-selector/issues"
+  },
+  "homepage": "https://github.com/near/wallet-selector/tree/main/packages/fastauth-metamask",
+  "peerDependencies": {
+    "near-api-js": "^0.44.2 || ^1.0.0"
+  }
```

Comment:
We have recently bumped the version for `near-api-js` to `v2.1.0`.

Can you please merge your branch with the latest `dev` branch?

Also the range will need to be updated here to:

```json
  "peerDependencies": {
    "near-api-js": "^1.0.0 || ^2.0.0"
  }
```

This is the PR which addressed the version change of NAJ it might be helpful to speed up your process: https://github.com/near/wallet-selector/pull/776